Dockerfile.client

```diff
@@ -9,7 +9,7 @@ RUN go mod download && go mod verify
 
 COPY . .
 
-WORKDIR /usr/src/app/client/cache_client
```

Comment:
Can you change the package name to `client`? I want to follow the go package naming conventions: https://go.dev/blog/package-names

Note that you will also need move the .go files out of `cache_client/` up a directory and then delete the `cache_client` directory